test/jdk/java/io/Serializable/serialFilter/SerialFilterFunctionTest.java

```diff
@@ -36,12 +36,19 @@
 import static java.io.ObjectInputFilter.Status.UNDECIDED;
 
 /* @test
- * @run testng/othervm SerialFilterFunctionTest
+ * @run testng/othervm -Djava.util.logging.config.file=${test.src}/logging.properties
+ *                      SerialFilterFunctionTest
  * @summary ObjectInputFilter.Config Function Tests
  */
 @Test
 public class SerialFilterFunctionTest {
 
+    static {
+        // Enable logging
+        System.setProperty("java.util.logging.config.file",
+                System.getProperty("test.src", ".") + "/logging.properties");
```

Comment:
Is `System.setProperty()` needed if it's already being set with -D ?

Reply:
Fooey; belt and suspenders not needed here.
In some cases, I used the static initializer to avoid adding command line arguments that would
obscure the command line arguments being tested.
In this case, the log may be interesting for debugging but not integral to the test correctness.